### PR TITLE
Update enterprise version

### PR DESCRIPTION
**Pull request by Chengxuan**
Pulls in the latest hyper ledger main into enterprise branch

### CODE REVIEW

**Comment by EnriqueL8** [comment]
There is quite a lot in this PR and it's basically updating it with FF 1.3

File: Dockerfile [hunk 1]
```diff
@@ -48,15 +66,24 @@ RUN trivy sbom /sbom.spdx.json --severity UNKNOWN,HIGH,CRITICAL --exit-code 1
...
     && chmod +x ./migrate \
     && mv ./migrate /usr/bin/migrate
-COPY --from=firefly-builder /firefly/firefly ./firefly
-COPY --from=firefly-builder /firefly/db ./db
-COPY --from=solidity-builder /firefly/solidity_firefly/build/contracts ./contracts
-COPY --from=fabric-builder /firefly/smart_contracts/fabric/firefly-go/firefly_fabric.tar.gz ./contracts/firefly_fabric.tar.gz
+COPY --from=firefly-builder --chown=1001:0 /firefly/firefly ./firefly
+COPY --from=firefly-builder --chown=1001:0 /firefly/db ./db
+COPY --from=solidity-builder --chown=1001:0 /firefly/solidity_firefly/build/contracts ./contracts
+COPY --from=fabric-builder --chown=1001:0 /firefly/smart_contracts/fabric/firefly-go/firefly_fabric.tar.gz ./contracts/firefly_fabric.tar.gz
```
Comment (EnriqueL8) [inline Dockerfile:86]: Need to be careful with this one and the way we run on k8s

**Review by EnriqueL8 — COMMENTED** [review]
I'm happy with bringing these changes in but needs thorough testing in dev, especially the Dockefile changes

**Review by nguyer — APPROVED** [review]
Leaving my approval on this, as all of this has presumably already been reviewed in the open source.

https://github.com/hyperledger/firefly/pull/1461/files#diff-b2e77d2a6574b8e68136b226be7c14dff6897e0c30751d33585d9d2bb916cea9R25-R42 might be a very relevant bit of the docs that anyone running this themselves should be aware of

**Pull request merged**